packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -389,6 +436,7 @@ export class AndroidEmulatorDevice extends DeviceBase {
       await this.configureMetroPort(build.packageName, metroPort);
       await this.launchWithBuild(build);
     }
+    this.mirrorNativeLogs(build);
```

Comment:
given launch adb commands are async, is there a chance that the process would launch before we start listening to "Start proc" log? Maybe we should call this after `force-stop` but before the launch command is triggered?